students_projects.html

```diff
@@ -40,7 +40,7 @@ <h1> <strong>Student Projects</strong> </h1>
                                <hr class="student-project-hr">
                                <div class="student-project-footer-container">
                                   <img src="https://github.com/{{project.github}}.png" class="img-circle student-project-creator-img">
-                                  <a href="https://github.com//{{project.github}}" target="_self" class="pull-right">{{project.creator}}</a>
+                                  <a href="https://github.com//{{project.github}}" target="_blank" class="pull-right">{{project.creator}}</a>
```

Comment:
@PulkitMishra  This should be _self, as the project should open in the same frame.

Reply:
@sk9331657 done. Thanks for pointing that I missed it out.